### DISCUSSION TITLE
How does this compare to `github.com/piersdeseilligny/betterfountain`?

### DISCUSSION THREAD

**Discussion by beedell-roke** [oparaskos/vscode-fountain#39] [Q&A] [answered]
Specifically https://github.com/piersdeseilligny/betterfountain/blob/b869bee1cf0365e19bccc3458389c4838044b7d8/README.md#features.

**Comment by oparaskos** [comment, marked as answer]
Short answer is that his is more mature and probably more stable, and prettier.

Long answer is;

There're some things that I want to do with this in the longer-term (time permitting) that didnt seem to fit extending his work directly.

This is also kind of a test-bed for writing a fountain parser that breaks the document down into more of a tree structure, and with less reliance on regex.

This has code lenses which IIRC his does not.

The language server is also seperable from the vscode plugin, so in theory you can also use this with another editor.
